packages/lesswrong/components/users/KarmaChangeNotifier.tsx

```diff
@@ -167,7 +168,9 @@ const KarmaChangesDisplay = ({karmaChanges, classes, handleClose }: {
             ))}
             {karmaChanges.comments && karmaChanges.comments.map(commentChange => (
               <MenuItemUntyped className={classes.votedItemRow}
-                component={Link} to={commentGetPageUrlFromIds({postId:commentChange.postId, postSlug:commentChange.postSlug, tagSlug:commentChange.tagSlug, commentId: commentChange._id})} key={commentChange._id}
+                // tagCommentType is given a String type in packages/lesswrong/lib/collections/users/karmaChangesGraphQL.ts because we couldn't get an inline union of literal types to work,
+                // but actually we know it will always be a TagCommentType because the db schema contstrains it
+                component={Link} to={commentGetPageUrlFromIds({postId:commentChange.postId, tagSlug:commentChange.tagSlug, tagCommentType:commentChange.tagCommentType as TagCommentType, commentId: commentChange._id})} key={commentChange._id}
```

Comment:
We removed postSlug because it it in fact always null

Reply:
s/null/undefined/